### DISCUSSION TITLE
WithFileUploads is not working with GCS and Google App Engine / Laravel 8 with Livewire update profile photo error

### DISCUSSION THREAD

**Discussion by manunoly** [livewire/livewire#4318] [1. Help]
Laravel 8 with Livewire update profile photo get error

I config livewire to use GCS
![image](https://user-images.githubusercontent.com/636156/144053451-de1071ab-de8a-4f8f-8c9a-875f79fa4da1.png)

here the filesystems.php config file
![image](https://user-images.githubusercontent.com/636156/144054264-e1cee413-1d63-4c83-a606-11191d16e569.png)

WithFileUploads contains some settings and validations for *** s3 ***
![image](https://user-images.githubusercontent.com/636156/144053633-fee7cb7a-30ec-4d85-92c0-ee11a26ad508.png)

but uploading a new profile picture ends in error
![image](https://user-images.githubusercontent.com/636156/144057252-d8764e64-92a5-4439-ada5-c7224586cd33.png)

*"ErrorException: file_put_contents(/tmp/www/html/framework/cache/facade-c2f53e2d655810ac111b48d42b34b209861bb985.php): failed to open stream: No such file or directory in /workspace/vendor/laravel/framework/src/Illuminate/Foundation/AliasLoader.php:108
Stack trace:
#0 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'file_put_conten...', '/workspace/vend...', 108, Array)
#1 /workspace/vendor/laravel/framework/src/Illuminate/Foundation/AliasLoader.php(108): file_put_contents('/tmp/www/html/f...', '<?php\n\nnamespac...')
#2 /workspace/vendor/laravel/framework/src/Illuminate/Foundation/AliasLoader.php(92): Illuminate\Foundation\AliasLoader->ensureFacadeExists('Facades\\Livewir...')
#3 /workspace/vendor/laravel/framework/src/Illuminate/Foundation/AliasLoader.php(74): Illuminate\Foundation\AliasLoader->loadFacade('Facades\\Livewir...')
#4 [internal function]: Illuminate\Foundation\AliasLoader->load('Facades\\Livewir...'*

AliasLoader.php is always looking for *'framework/cache/facade-'* path
![image](https://user-images.githubusercontent.com/636156/144054940-e325b8e9-b79d-47df-9401-bd9d087c5ea7.png)

some help will be good to use the WithFileUploads with Google App Engine

**Comment by joshhanley** [comment]
@manunoly I've not used google cloud storage, so can't be of help, but everything that is required for it to run has been previously PR'd https://github.com/livewire/livewire/pull/3198 and https://github.com/livewire/livewire/pull/3499

So might be worth having a look at them and see if you can get it running from that.

Hope this helps!

**Comment by protoapostoli** [comment]
Was this ever solved? I'm getting the same error on GAE.

**Comment by protoapostoli** [comment]
For anyone who stumbles upon this later, here was my solution:

https://github.com/livewire/livewire/discussions/5719
